modules/graphql_1/main.tf

```diff
@@ -185,14 +177,14 @@ resource "aws_ecs_task_definition" "gql_task" {
   requires_compatibilities = ["FARGATE"]
   cpu                      = "1024"
   memory                   = "2048"
-  task_role_arn            = aws_iam_role.orca_ecs_tasks_role.arn
+  task_role_arn            = var.gql_tasks_role_arn
   execution_role_arn       = aws_iam_role.orca_ecs_task_execution_role.arn
   tags                     = var.tags
   container_definitions    = <<DEFINITION
 [
   {
     "name": "orca-gql",
-    "image": "ghcr.io/nasa/cumulus-orca/graphql:0.0.20",
+    "image": "ghcr.io/nasa/cumulus-orca/graphql:0.0.21",
```

Comment:
we can also make this a variable in terraform

Reply:
We could, but I don't think we'd want customers adjusting this, and it isn't used anywhere else.

Reply:
makes sense. We just have to remember to update the version here in the future.

Reply:
Yup. We'll need to do that no matter how we set it up.